### DISCUSSION TITLE
关于hexo d失败

### DISCUSSION THREAD

**Discussion by wchyin** [YunYouJun/hexo-theme-starry#105] [Q&A 疑难杂症] [answered]
这是我的另一个hexo博客，按照跟以前一样的方式（后来也对过教程）设置了deploy，但是每一次`hexo d`都没效果
[![ywmq54.png](https://s3.ax1x.com/2021/02/10/ywmq54.png)](https://imgchr.com/i/ywmq54)
求助求助 以下为hexo版本
[![ywmXG9.png](https://s3.ax1x.com/2021/02/10/ywmXG9.png)](https://imgchr.com/i/ywmXG9)

**Comment by YunYouJun** [comment, marked as answer]
在 `hexo deploy` 前，你应当 `hexo g` 生成静态文件。具体请参见 [hexo 文档](https://hexo.io/zh-cn/docs/commands#generate)。

**Reply by wchyin** [reply]
我已经生成了

**Reply by YunYouJun** [reply]
请展示生成日志，并确保已在正确位置 clone 主题。

**Reply by wchyin** [reply]
我现在是只能靠手动上传

**Reply by wchyin** [reply]
一切都是正常的

**Reply by wchyin** [reply]
[![ywrd0K.png](https://s3.ax1x.com/2021/02/10/ywrd0K.png)](https://imgchr.com/i/ywrd0K)

**Reply by YunYouJun** [reply]
这可能与新版 [#4634 | hexo](https://github.com/hexojs/hexo/issues/4634) BUG 有关。

一个临时的解决方案是：

```bash
npm un hexo-deployer-git
npm i hexojs/hexo-deployer-git
```

此外的建议是，借此机会不妨考虑一下 CI 等集成部署方案。

> https://hexo.io/zh-cn/docs/one-command-deployment#Vercel

**Reply by Lfhsheng** [reply]
> 我现在是只能靠手动上传

可以用cmd试一试
